frontend/.gitignore

```diff
@@ -0,0 +1,43 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
Why there are two ```.gitignore``` files - one in main dir and one in frontend ?

Reply:
Changed to one main `.gitignore` file and two separate, more specific `.gitingnore` files for `backend` and `frontend`.